doc/sources/daal4py.rst

```diff
@@ -0,0 +1,1224 @@
+.. Copyright contributors to the oneDAL project
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+.. include:: substitutions.rst
+.. _daal4py_ref:
+
+daal4py API Reference
+=====================
+
+This is the full documentation page for ``daal4py`` functions and classes. Note that for the most
+part, these are simple wrappers over equivalent functions and methods from the |onedal|. See also
+the `documentation of oneDAL <https://uxlfoundation.github.io/oneDAL/>`__ for more details.
```

Comment:
It would be better to mention that this is the wrapper over the specific part of the interfaces.
Something like this:
```suggestion
the `documentation of DAAL interfaces <https://uxlfoundation.github.io/oneDAL/daal-interfaces.html>`__ for more details.
```